src/cmd/kssh/kssh.go

```diff
@@ -35,12 +34,18 @@ func main() {
 		doAction(action, keyPath, remainingArgs)
 		os.Exit(0)
 	}
-	config, err := getConfig(team)
+
+	bot, err := kssh.NewBot()
```

Comment:
how this works:
first need to set up go chat bot
then in `Configure()`, need to use the bot api to get the kssh config from the KV store, to return a `ConfiguredBot`